billcom_integration/models/account_payment.py

```diff
@@ -0,0 +1,1023 @@
+            ("WALLET", "BILL Balance"),
+            ("AP_CARD", "AP Card"),
+        ],
+        string="Funding Account Type",
+        default="BANK_ACCOUNT",
+        help="Type of funding account for Bill.com payment",
+    )
+    billcom_process_date = fields.Date(
+        string="Process Date",
+        help=(
+            "Date when Bill.com will process this payment (format: YYYY-MM-DD). "
+            "Required for WALLET and AP_CARD funding types. "
+            "For new vendor bank accounts, must be at least 2 business days from today."
+        ),
+    )
+    is_process_date_sync = fields.Boolean(compute="_compute_process_sync_date")
+    billcom_pay_faster = fields.Boolean(
+        string="Pay Faster",
+        default=False,
+        help="Enable Pay Faster for expedited payment delivery",
+    )
+    billcom_check_delivery_type = fields.Selection(
+        selection=[
+            ("STANDARD", "Standard"),
+            ("RTP_DELIVERY", "Real-Time Payment (ACH)"),
+            ("UPS_1DAY", "UPS 1-Day Delivery"),
+            ("UPS_2DAY", "UPS 2-Days Delivery"),
+            ("UPS_3DAY", "UPS 3-Days Delivery"),
+            ("USPS_PRIORITY", "USPS Priority"),
+        ],
+        string="Check Delivery Type",
+        default="STANDARD",
+        help="Delivery method for check payments",
+    )
+    billcom_payment_status = fields.Selection(
+        [
+            ("draft", "Draft"),
+            ("scheduled", "Scheduled"),
+            ("processing", "Processing"),
+            ("processed", "Processed"),
+            ("sent", "Sent"),
+            ("canceled", "Canceled"),
+            ("failed", "Failed"),
+        ],
+        string="Bill.com Payment Status",
+        readonly=True,
+        copy=False,
+        default="draft",
+    )
+    is_international_payment = fields.Boolean(
+        string="International Payment",
+        compute="_compute_is_international_payment",
+        store=True,
+    )
+    billcom_confirmation_number = fields.Char(
+        string="Bill.com Confirmation Number", readonly=True, copy=False
+    )
+    billcom_transaction_number = fields.Char(
+        string="Bill.com Transaction Number", readonly=True, copy=False
+    )
+    billcom_exchange_rate = fields.Float(
+        string="Exchange Rate", digits=(16, 6), readonly=True
+    )
+    billcom_funding_amount = fields.Monetary(
+        string="Funding Amount (USD)", readonly=True
+    )
+    reconciled_bill_ids = fields.Many2many(
+        comodel_name="account.move",
+        relation="account_payment_bill_rel",
+        column1="payment_id",
+        column2="bill_id",
+        string="Reconciled Bills",
+        help="Bills reconciled with this payment for Bill.com sync",
+        domain="[('move_type', '=', 'in_invoice')]",
+    )
+
+    @api.depends("partner_bank_id.billcom_last_sync_date")
+    def _compute_process_sync_date(self):
+        for rec in self:
+            if self.partner_bank_id.billcom_last_sync_date:
```

Comment:
```suggestion
            if rec.partner_bank_id.billcom_last_sync_date:
```
I think you missed here